### ISSUE TITLE
Request: improve accessibility for blind people

### ISSUE THREAD

**Issue by cedricgr** [keybase/client#10370]
Dear Keybase team:

I have, in one of the newly-supported Teams that I'm responsible for, a blind user. He is reporting several issues regarding lack of accesibility standards to blind people, such as:
- lack of description/caption for clickable buttons and other items
- lack of adherence to accessibility standards

He reports he had to randomly click everything in order to discover how the Keybase client works.
